lib/repo_tools.pm

```diff
@@ -253,6 +349,14 @@ sub disable_source_repo {
     }
 }
 
+
+=head2 generate_version
+
+ generate_version([$separator]);
+
+Generate SLE or openSUSE versions. C<[$separator]> is separator used for version number, it will be default to _ if omitted. Example: SLES-12-4, openSUSE_Leap
```

Comment:
The `[` and `]` are not part of the var name and are just indicating that the parameter is optional.

```suggestion
Generate SLE or openSUSE versions. C<$separator> is separator used for version number, it will be default to _ if omitted. Example: SLES-12-4, openSUSE_Leap
```

Reply:
yes, changed now